cmd/mcp-server/eval/scripts/score-results.py

```
@@ -143,8 +152,56 @@ def main():
+            if not runs:
+                continue
+            combined = {}
+            for key in runs[0]:
+                vals = [r[key]["value"] for r in runs if key in r and r[key].get("value") is not None]
+                rats = [r[key].get("rationale", "") for r in runs if key in r and r[key].get("value") is not None]
+                if not vals:
+                    combined[key] = {"value": None, "rationale": ""}
+                elif isinstance(vals[0], bool):
+                    passes = sum(1 for v in vals if v)
+                    winner = passes > len(vals) / 2
+                    rat = next((r for v, r in zip(vals, rats) if v == winner), "")
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)